README.md

```diff
@@ -1,6 +1,7 @@
 # Exabyte Scientific Projects
 
-This repository contains various scientific projects run by Exabyte.io
+This repository contains various scientific projects run by Exabyte.io. Create a virtual environment, and install the
```

Comment:
Long as we're using Python 3+, users aren't going to run the type of compatibility issues that the 2.7->3 leap introduced. Python versions at this point are really just "Use at _least_ this version" in case a new feature got used. I've added the version (3.8.6) to the Digital Ecosystem folder's readme.

Reply:
Let's move requirements there too then - into the subfolder. And add actual steps to go through - like in https://github.com/Exabyte-io/express#installation

Reply:
Moved it into the digitalEcosystem folder and updated the readme accordingly